_topic_map.yml

```diff
@@ -67,6 +67,8 @@ Topics:
     File: installing-aws-default
   - Name: Installing a cluster on AWS with customizations
     File: installing-aws-customizations
+  - Name: Installing a cluster on AWS with network customizations
+    File: installing-aws-network-customizations
```

Comment:
Virtually all of the network customizations must be made at install time and can't be changed afterward.

Reply:
That's fair! Do you think most users will want to make these changes?

Reply:
"Most"? No

Actually, I guess for people installing on AWS, changing the network defaults will be fairly uncommon. (People installing on bare metal or VMware or OpenStack are more likely to have problems with the default values.)

Reply:
Thanks! Then I think it does need to be a separate assembly. We're isolating different installation user stories in separate assemblies so that users have an easier time finding the information that they need. (If you'd said that most users would want to tinker with the network settings in AWS, then I would vote to combine the assemblies.)